### PR TITLE
Setup CI/CD for wallet Javascript bindings 

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: .github/workflows/js_publish.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: Javascript wallet
```
Comment (minikin) [inline .github/workflows/js_publish.yml:1]: I'd suggest having an explicit name for the  job, something like **"Release JavaScript Wallet"** and a workflow file name like `publish_js_wallet.yml`

File: .github/workflows/js_publish.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+on:
+  push:
+    branches: [ "main" ]
```
Comment (minikin) [inline .github/workflows/js_publish.yml:5]: Do we really want to trigger this workflow on each PR merge?

File: .github/workflows/js_tests.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+name: Javascript wallet
```
Comment (minikin) [inline .github/workflows/js_tests.yml:1]: I'd also suggest having an explicit name for the job, something like "JavaScript Wallet Tests." and the same for the workflow file name.

File: .github/workflows/js_tests.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    - uses: jetli/wasm-pack-action@v0.3.0
+      with:
+        # Optional version of wasm-pack to install(eg. 'v0.9.1', 'latest')
```
Comment (minikin) [inline .github/workflows/js_tests.yml:22]: Can we remove this comment? It doesn't add any value here, IMO.

**Review by minikin — APPROVED** [review]
Approved! I left a few minor comments.

File: .github/workflows/js_publish.yml (hunk 2 above)
Comment (Mr-Leshiy) [reply .github/workflows/js_publish.yml]: It will only work after we have establish `tag`

**Pull request merged**